Tests/Common/Securities/Equity/PatternDayTradingMarginModelTests.cs

```diff
@@ -0,0 +1,221 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Linq;
+using NUnit.Framework;
+using QuantConnect.Data;
+using QuantConnect.Data.Market;
+using QuantConnect.Indicators;
+using QuantConnect.Orders;
+using QuantConnect.Securities;
+using QuantConnect.Securities.Equity;
+
+namespace QuantConnect.Tests.Common.Securities.Equity
+{
+    [TestFixture]
+    public class PatternDayTradingMarginModelTests
+    {
+        private static readonly DateTime Noon = new DateTime(2016, 02, 16, 12, 0, 0);
+        private static readonly DateTime MidNight = new DateTime(2016, 02, 16, 0, 0, 0);
+        private static readonly DateTime NoonWeekend = new DateTime(2016, 02, 14, 12, 0, 0);
+        private static readonly DateTime NoonHoliday = new DateTime(2016, 02, 15, 12, 0, 0);
+
+        private static readonly TimeKeeper TimeKeeper = new TimeKeeper(Noon.ConvertToUtc(TimeZones.NewYork),
+            TimeZones.NewYork);
+
+        private static Security CreateSecurity(DateTime newLocalTime)
+        {
+            var security = new Security(CreateUsEquitySecurityExchangeHours(), CreateTradeBarConfig());
+            security.Exchange.SetLocalDateTimeFrontier(newLocalTime);
+            security.SetLocalTimeKeeper(TimeKeeper.GetLocalTimeKeeper(TimeZones.NewYork));
+            security.SetMarketPrice(new IndicatorDataPoint(Symbols.SPY, newLocalTime, 100m));
+            return security;
+        }
+
+        private static SecurityExchangeHours CreateUsEquitySecurityExchangeHours()
```

Comment:
In resharper's settings you can set the order of the methods, by convention we push private methods to the bottom. This link should be able to guide you on setting up your local resharper to move private methods to the bottom, you should be able to use the graphic interface and not need to modify the XML directly. I'll look into getting some solution specific settings in so everyone is using the same resharper dotSettings file